core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/configure-postgresql.yml

```diff
@@ -85,6 +80,17 @@
     backup: yes
   register: change_postgresql_conf
 
+# Placed here to avoid errors in log due to nonexistent directory
```

Comment:
What kind of errors? Which step generates this errors?
I know it's not part of this task to make refactoring of whole role, but i think we don't need to archive WAL logs since we don't use them in any part of our code (i didn't find any reference). But it's good step to start with Point-in-Time-Recovery solution for Postgres (which we  currently don't support). AFAIK repmgr uses psql to connect to database and replicate data.
OK, maybe we should leave it like it is, since it's backport and clean this in develop and maybe LTS since it can unnecessary fill up disk space.

Reply:
This task will be removed in latest versions with PR #2311. It can be removed here, but not sure that we need to do that for backport.